src/interfaces/node.h

```diff
@@ -248,12 +251,12 @@ class Node
 
     //! Register handler for block tip messages.
     using NotifyBlockTipFn =
-        std::function<void(bool initial_download, int height, int64_t block_time, double verification_progress)>;
+        std::function<void(bool initial_download, const uint256 blockHash, int height, int64_t block_time, double verification_progress)>;
```

Comment:
In commit "Added best block hash to the NotifyHeaderTip and NotifyBlockTip signals." (4948d4ee50d9960b23f43e14938845b0bb21772b)

Maybe use reference type `const uint256&` to be a little more efficient and avoid a copy here and below